rosetta-lang/src/main/java/com/regnosys/rosetta/generator/RosettaGenerator.xtend

```diff
@@ -146,10 +148,13 @@ class RosettaGenerator implements IGenerator2 {
 						Data: {
 							dataGenerator.generate(packages, fsa, it, version)
 							metaGenerator.generate(packages, fsa, it, version)
+							//Legacy
 							validatorsGenerator.generate(packages, fsa, it, version)
 							it.conditions.forEach [ cond |
 								conditionGenerator.generate(packages, fsa, it, cond, version)
 							]
+							//new
+							//validatorGenerator.generate(packages, fsa, it, version)
```

Comment:
Shouldn't this one be uncommented?

Reply:
if we use then I think it will call our new code which we don't want right now